src/app/pages/network/interfaces/interfaces-form/interfaces-form.component.ts

```diff
@@ -337,22 +263,41 @@ export class InterfacesFormComponent implements OnDestroy {
     this.vlan_pint = _.find(this.fieldConfig, {'name' : 'vlan_parent_interface'});
     this.bridge_members = _.find(this.fieldConfig, {'name' : 'bridge_members'});
     this.lag_ports = _.find(this.fieldConfig, {'name' : 'lag_ports'});
-    this.route.params.subscribe(params => {
-      if(!params['pk']) {
-        this.type.type = 'select';
-      } else {
-        this.iparrayControl.initialCount = this.initialCount['aliases']
-          = this.initialCount_default['aliases'] = 0;
 
-        this.iparrayControl.formarray[3]['isHidden'] = false;
-        this.iparrayControl.formarray[3].disabled = false;
-      }
-    });
+    if (window.localStorage.getItem('is_freenas') === 'false') {
+      this.ws.call('failover.node').subscribe((node) => {
+        if (node === 'A') {
+          this.ipPlaceholder = ` (${globalHelptext.thisCtlr})`;
+          this.failoverPlaceholder = ` (${globalHelptext.Ctrlr} 2)`;
+        } else if (node === 'B') {
+          this.ipPlaceholder = ` (${globalHelptext.Ctrlr} 1)`;
+          this.failoverPlaceholder = ` (${globalHelptext.thisCtlr})`;
+        } else {
+          this.ipPlaceholder = ' The active controller cannot be detected.';
```

Comment:
Does this need a T() for translation?  Should it be in the global helptext file?

Reply:
done